src/api/ResourcefulApi.js

```diff
@@ -79,7 +79,13 @@ export class ResourcefulApi extends Api {
   setupApiModules (apiModulesToRegister = []) {
     Performance.mark('api_setup_modules_start')
 
-    apiModulesToRegister.forEach(moduleName => this.registerModule(moduleName, this[moduleName]))
+    apiModulesToRegister.forEach(moduleName => {
+      let resourceProxy = this[moduleName]
+      if (!resourceProxy) {
+        resourceProxy = new ResourceProxy()
+      }
```

Comment:
we may move this check to the ModuleBuilder. This would affect a couple of occurences where we added this the last dasy. But i am not sure about the arcitecture here.

Reply:
I will have a look into this later, for now this is fine.